src/components/layout/Footer/index.js

```diff
@@ -32,7 +32,12 @@ const Footer = () => (
     </SitemapSection>
     <ContactSection>
       <CapsHeadingSmall>contact</CapsHeadingSmall>
-      <EmailLink>hello@yallacooperative.com</EmailLink>
+      <EmailLink
+        href="mailto:hello@yallacooperative.com?Subject=Hello%20again"
+        target="_top"
```

Comment:
use the `rel`? security i think, i dont think it matters personally

Reply:
yea I think here its fine to not use it